main.go

```diff
@@ -0,0 +1,334 @@
+package main
+
+import (
+    "bytes"
+
+    "fmt"
+
+    "path/filepath"
+
+    "os"
+    "os/exec"
+    "os/user"
+
+    "io/ioutil"
+
+    "github.com/drone/drone-plugin-go/plugin"
+)
+
+var (
+    buildCommit string
+)
+
+var netrcFile = `
+machine %s
+login %s
+password %s
+`
+
+type Params struct {
+    UpstreamName    string            `json:"upstream"`
+    PagesDirectory  string            `json:"pagesDirectory"`
+    TemporaryBase   string            `json:"temporaryBaseDirectory"`
```

Comment:
let's use snake case for the names. I would also suggest some minor name changes just for consistency with other plugins:

```
branch: gh-pages 
source:  dist/
temp: /tmp/gh-pages
```